phoenicis-containers/pom.xml

```diff
@@ -64,5 +64,10 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.phoenicis</groupId>
+            <artifactId>phoenicis-library</artifactId>
+            <version>5.0-SNAPSHOT</version>
```

Comment:
We will have a circular dependency problem here, since later, we'll also want to add a configure button on the shortcut